templates/zerver/base.html

```diff
@@ -19,6 +19,9 @@
     <link href="/static/favicon.ico?v=2" rel="shortcut icon">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <link href="/static/node_modules/katex/dist/katex.min.css" rel="stylesheet">
+    <script src="/static/node_modules/katex/dist/katex.min.js"></script>
```

Comment:
Yep, they are provided!  That's basically the cool thing about PIPELINE/JS_SPECS; the thing will work magically in both dev and prod :)